OOPTask/Rectangle.java

```diff
@@ -1,10 +1,14 @@
-public class Rectangle extends FiguresWithSides{
-    private String name = "Rectangle";
+package OOPTask;
+
+import Utils.FiguresWithSides;
+
+public class Rectangle extends FiguresWithSides {
+    private String name = "OOPTask.Rectangle";
     private int length;
     private int width;
     final int NUMBER_OF_SIDES = 4;
```

Comment:
You have different getters for this constant in Rectangle and Square.

Reply:
Fixed it, Thank You!